tests/helpers_tests/test_condition.py

```diff
@@ -108,6 +119,134 @@ def test_evaluate__returns_null__when_condition_has_an_invalid_type_property(sel
 
         self.assertIsNone(evaluator.evaluate(0))
 
+    def test_evaluate__returns_true__when_user_version_matches_target_version(self):
+
+        evaluator = condition_helper.CustomAttributeConditionEvaluator(
+            eq_semver_condition_list_variation_1, {'Android': '2.0'}, self.mock_client_logger
+        )
+
+        self.assertStrictTrue(evaluator.evaluate(0))
+
+        evaluator = condition_helper.CustomAttributeConditionEvaluator(
+            eq_semver_condition_list_variation_1, {'Android': '2.0.0.0'}, self.mock_client_logger
+        )
+
+        self.assertStrictTrue(evaluator.evaluate(0))
+
+        evaluator = condition_helper.CustomAttributeConditionEvaluator(
+            eq_semver_condition_list_variation_2, {'Android': '2.0.0'}, self.mock_client_logger
+        )
+
+        self.assertStrictTrue(evaluator.evaluate(0))
+
+        evaluator = condition_helper.CustomAttributeConditionEvaluator(
+            eq_semver_condition_list_variation_3, {'Android': '2.0.1-beta'}, self.mock_client_logger
+        )
+
+        self.assertStrictTrue(evaluator.evaluate(0))
+
+    def test_evaluate__returns_true__when_user_version_does_not_match_target_version(self):
```

Comment:
returns_true, interesting... 
what's the difference between the following, both returning true. 
`test_evaluate__returns_true__when_user_version_does_not_match_target_version`
`test_evaluate__returns_true__when_user_version_matches_target_version`